### PR TITLE
set up associations for entities and edit initDb to load in properly

### PR DESCRIPTION
**Pull request by SamsonDeVol**
Pretty large PR so would be good for everyone to look at it before merging. I set up all the associations for entity relationships so that sequalize can initialize the database without any errors

### CODE REVIEW

**Review by jonahbroyer — APPROVED** [review]
Looks good!

**Review by isnide23 — APPROVED** [review]
Everything looks fine to me, should be good to merge

File: data/courses.json
Comment (swearingenmj) [inline data/courses.json:1]: Why did you remove instructorId from the courses data?

File: data/users.json
Comment (swearingenmj) [inline data/users.json:1]: Should we add more than one user to the data? Possibly have at least one user for each role? That way we don't have to change the user's role.

File: models/submission.js
Comment (swearingenmj) [inline models/submission.js:1]: Do we not need foreign keys in submission? Does having foreign keys in submission AND the other models create a redundancy?

**Review by swearingenmj — CHANGES_REQUESTED** [review]
See comments. Let's talk before merging.

File: data/courses.json
Comment (SamsonDeVol) [reply data/courses.json:1]: instructorId was referenecing users that don't exist. Once we have a more robust idea of our models, me or someone else could go back and make more complete example data

File: data/users.json
Comment (SamsonDeVol) [reply data/users.json:1]: Yeah we should!

Comment (swearingenmj) [reply data/users.json:1]: added issue to resolve this in future PRs

File: data/courses.json
Comment (swearingenmj) [reply data/courses.json:1]: added issue to resolve in future PRs

**Review by swearingenmj — APPROVED** [review]
